package.json

```diff
@@ -59,7 +59,7 @@
     "babel-gettext-extractor": "^1.0.2",
     "babel-loader": "5.3.3",
     "chai": "3.4.1",
-    "enzyme": "^1.1.0",
+    "enzyme": "2.1.0",
```

Comment:
I  use an Enzyme utility method in this patch (`childAt`) that was added in 2.1.0.